### PR TITLE
EI S05: fade to silence when recounting the priestess's death (v2)

### PR DESCRIPTION
**Pull request by Dalas121**
In EI's S05, Owaec briefly recounts the bloody fate of his local priestess, killed by undead.

The scenario's music consists of wanderer.ogg and traveling_minstrels.ogg, which are inappropriate for encountering a mangled corpse.

Instead, fade the music out during this section.

This is the second version of this pull request; I did something weird on the first one causing the entire file to change.

### CODE REVIEW

File: data/campaigns/Eastern_Invasion/scenarios/05_Northern_Outpost.cfg [hunk 1]
```diff
@@ -193,6 +193,30 @@
     {PLACE_IMAGE items/bones.png 6 31}
     {PLACE_IMAGE scenery/blood-1.png 7 31}
 
+#define CHANGE_VOLUME VOLUME DELAY
+    [volume]
+        music={VOLUME}
+    [/volume]
+    [delay]
+    time={DELAY}
```
Comment (irydacea) [inline data/campaigns/Eastern_Invasion/scenarios/05_Northern_Outpost.cfg:201]: Bad indent here, see the [CI logs](https://github.com/wesnoth/wesnoth/actions/runs/8271100101/job/22630101946#step:6:23).

**Comment by irydacea** [comment]
(Previously discussed in #8536.)

**Comment by Dalas121** [comment]
Fixed, thanks

**Comment by nemaara** [comment]
It's a bit of a workaround but I'd rather have neutral to solemn tone background tracks for this scenario rather than fading out the music entirely. What if we just changed the tracks completely? Say Weight of Revenge and Nunc Dimittis. They're not perfect as they fit better with other scenarios, but that's a lack of general solemn mood compared to battle music that we have.

**Comment by Dalas121** [comment]
From the old issue:

> I understand where you're coming from, but "Wanderer" and "Traveling Minstrels" were deliberate choices for S05, because that's how I had intended the scenario's atmosphere. After the caustic fumes of the Bitter Swamp and the deadly terrors of Mal Ravanal's Capitol, you finally return to familiar terrain. Instead of fighting necromancers and liches, you fight a few bandits and a rather silly dark adept, neither of whom pose much of a threat either narratively or mechanically.
> 
> And against all odds, Owaec has even managed to survive, along with most of the villagers! Some died, tragically I'm sure, but the overall tone of the scenario was supposed to be "the undead are really dangerous, but at least we can catch our breaths here", which is why I chose the music in question.

Not a hill I'm willing to die on, but that was the original intent.

**Comment by nemaara** [comment]
Yea that's why I was suggesting a more neutral track (traveling minstrels sounds kinda happy to me, wanderer not entirely appropriate) but it doesn't seem like we have a lot. One of these days I'll load up finale and write some music but maybe for now silence is okay or another option is play weight of revenge or nunc dimittis just once?

**Review by nemaara — APPROVED** [review]

**Comment by Dalas121** [comment]
Ah I see.  If you're looking for music, I got several of the pieces for TDG from [Scott Buckley](www.scottbuckley.com.au). All CC-BY 4.0 (which should be fine, right?)

**Comment by nemaara** [comment]
Yeah should be fine. Any suitable ones?

**Comment by stevecotton** [comment]
Is this waiting for new music, or is it ready to merge?

**Comment by Dalas121** [comment]
New music would be good, but I don't have time to find something suitable at the moment.  Can merge for now; later if I (or anyone) finds good music we can make another PR.

**Pull request merged**